README.md

```diff
@@ -1,6 +1,14 @@
 # Measure Repository
 
-A prototype implementation of a [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html) and associated frontend application.
+A prototype implementation of a [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html) and associated frontend application. This repository is a monorepo that consists of: 
+- [Measure Repository Service](https://github.com/projecttacoma/measure-repository/blob/main/service/README.md)
+  - Implements portions of the [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html) specification
+  - Acts as a FHIR server and shared source of truth for measures
+- [Measure Repository App](https://github.com/projecttacoma/measure-repository/blob/main/app/README.md)
+  - A prototype [Next.js](https://nextjs.org/) application that demonstrates potential interactions with a FHIR Measure Repository Service
+  - Includes a frontend, backend, and small database which acts as in-progress storage space for drafting measures
```

Comment:
I would change "drafting measures" to "draft measures and libraries" or "draft artifacts" since it holds both

Reply:
similarly, I think the libraries are in support of the measures, so maybe
```suggestion
  - Includes a frontend, backend, and small database which acts as in-progress storage space for drafting measures and associated artifacts
```

Reply:
I'm trying to leave it open ended in case we tackle measure reports or something... heh